app/views/comments/new.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'form'%>
```

Comment:
- [ ] I could not verify the display of this form because it was not showing even the data I added, from the image below, you will notice I added data but could not show on the UI, kindly work on what is blocking the view.
![image](https://github.com/AlierPM/BlogPress/assets/86200356/f751e26a-95a4-4193-887a-8da2047cbcdd)

Reply:
It is all working on my side @Olamarx ; check the screenshot;  if you mind please. Thank for the constructive suggestions.